protostar/starknet/cheatable_syscall_handler.py

```diff
@@ -142,6 +146,14 @@ def _call_contract(
         else:
             raise NotImplementedError(f"Unsupported call type {syscall_name}.")
 
+        contract_call_data = (int(str(request.function_selector)), calldata)
+        if self.cheatable_state.contract_calls.get(contract_address):
+            self.cheatable_state.contract_calls[contract_address].append(
+                contract_call_data
+            )
+        else:
+            self.cheatable_state.contract_calls[contract_address] = [contract_call_data]
+
```

Comment:
You can wrap this code with region commends to highlight Protostar code.

```py
# region Modified Starknet code.
...
# endregion

Reply:
[use region](https://github.com/software-mansion/protostar/pull/1060/commits/ba212ef03af2b6a0d240bc3da29e947c8e081bc2)